src/Svg/Surface/CPdf.php

```diff
@@ -3,10 +3,7 @@
  * A PHP class to provide the basic functionality to create a pdf document without
  * any requirement for additional modules.
  *
- * Extended by Orion Richardson to support Unicode / UTF-8 characters using
- * TCPDF and others as a guide.
- *
- * @author  Wayne Munro <pdf@ros.co.nz>
+ * @author  Wayne Munro
```

Comment:
why did you drop the email?
is it not valid anymore?

Reply:
It is not. The domain registration lapsed a number of years ago.

Reply:
Is there a github link we could use or something like that
That's useful for copyright reasons when someone needs to contact the author
Anyway, feel free to ignore my comment

Reply:
The original source was hosted at the domain attached to that email address, which is where the version bundled with Dompdf originated. The source was posted under a public domain license, which is what we've maintained for that library even with the changes made over the years.

The Cpdf source lives on at https://github.com/rospdf/pdf-php. The version bundled with Dompdf has been developed in isolation of that project. Just a quick perusal of the source shows how much the two projects have diverged over the years. If somebody needs to contact the author of the Dompdf version it would probably be best to contact this team.

I'm planning to split our version out into a separate project to make for easier maintenance, at which point I may tweak the project meta to reflect the provenance. There's also the possibility of merging with the original at some point in the future though there's been no discussion around something like that at this time.